clarity-vm-tsd.md

```diff
@@ -0,0 +1,89 @@
+# Clarity VM | Technical Specification Document
+
+# Executive Summary:
```

Comment:
These H1 headers below line 1 should all be at least H2 `##` or H3 if a subsection